doc/support-matrix.md

```diff
@@ -0,0 +1,50 @@
+| **IMU Hardware Timestamps**    | No                     | Yes                               | Yes (via DKMS or patch)                    | Yes (via LIBUVC, 2.26+)     | Yes (via LIBUVC, 2.26+) | Yes (via LIBUVC, 2.26+) | No (WIP)          | Yes            | Yes         | Yes         | Yes | Yes                | Yes                 | N/A            | N/A    | Yes     | No     | No   |
+| **Hardware Timestamps**        | Yes                    | Yes (requires script or DFMT INF) | Yes (via DKMS or patch or Kernel >= 4.18)  | Yes (via LIBUVC)            | Yes (via LibUVC)        | Yes                     | Yes               | Yes            | Yes         | Yes         | Yes | Yes                | Yes                 | Yes            | Yes    | Yes     | Yes    | Yes  |
+| **Global Timestamps**          | Yes                    | Yes (requires script or DFMT INF) | Yes (via DKMS or patch or Kernel >= 4.18)  | Yes (via LIBUVC)            | Yes (via LIBUVC)        | Yes                     | Yes               | Yes            | Yes         | Yes         | Yes | Yes                | Yes                 | Yes            | Yes    | Yes     | Yes    | Yes  |
+| **On-Chip Calibration**           | Yes (2.26+)            | Yes (2.26+)                       | Yes (2.26+)                                | Yes (2.26+)                 | Yes (2.26+)             | No                      | Yes (2.26+)       | Yes (2.26+)    | No          | No          | No  | No                 | No                  | No             | No     | No      | No     | No   |
+| **Firmware Update**            | Yes (2.24+)            | Yes (2.24+)                       | Yes (2.24+)                                | Yes (2.24+)                 | Yes (2.24+)             | Yes (2.24+)             | Yes (2.24+)       | Yes (2.24+)    | Yes (2.24+) | Yes (2.24+) | No  | No                 | No                  | No             | No     | No      | No     | No   |
+| **Multicamera Support**        | Not tested             | Yes                               | Yes                                        | Not tested                  | Not tested              | Not tested              | Not tested        | Yes            | Yes         | Yes         | Yes | Yes                | Yes                 | No             | No     | Yes     | N/A    | N/A  |
+
+### SR300 Notes:
+
+* Global Timestamp mechanism is not yet available for the SR300
+* D400 JSON presets are not applicable to SR300 devices, since they rely on completely different technology
+* Multicamera is technically supported similar to D400 but SR300 devices suffer from destructive interference, hence should not shared field of view
```

Comment:
Maybe - "(SR300) Multicamera is technically supported similar to D400 but ~~SR300 devices~~ suffer..."

...not share**~~d~~**...